examples/experimental/interview_openhands/llm_agent.py

```diff
@@ -303,6 +303,15 @@ def get_action_template(self, selected_actions: list[ActionType]) -> str:
     async def aact(self, message: AgentAction | Tick | Text) -> AgentAction:
         match message:
             case Text(text=text):
+                if "BrowserOutputObservation" in text:
```

Comment:
Handling large browser output in a very naive way for now. Planos to create a browse handler node